script.js

```diff
@@ -1,4 +1,254 @@
+  const hands = ["scissors", "paper", "stone"];
+  return hands[Math.floor(Math.random() * 3)];
+}
+
+// to decide on who won the game
+function playGame(player, computer) {
+  if (player == computer) {
+    return draw(player, computer);
+  } else if (player == "scissors") {
+    if (computer == "stone") {
+      return lose(player, computer);
+    } else if (computer == "paper") {
+      return win(player, computer);
+    }
+  } else if (player == "paper") {
+    if (computer == "scissors") {
+      return lose(player, computer);
+    } else if (computer == "stone") {
+      return win(player, computer);
+    }
+  } else if (player == "stone") {
+    if (computer == "paper") {
+      return lose(player, computer);
+    } else if (computer == "scissors") {
+      return win(player, computer);
+    }
+  } else {
+    return "Please enter only scissors,paper or stone";
+  }
+}
+
+// adds some cute emoji to messages
+function cuteSelector(input) {
+  if (input == "paper") {
+    return "🖐";
+  } else if (input == "scissors") {
+    return "✌";
+  } else if (input == "stone") {
+    return "✊";
+  }
+}
+
+//winning message
+function win(player, computer) {
+  winCount++;
+  playCount++;
+  lastWinner = userName;
+  return `You won! You chose ${player}${cuteSelector(
+    player
+  )} and the computer chose ${computer}${cuteSelector(
+    computer
+  )}. <br>${winRateMessage(winCount, playCount, userName)}`;
+}
+
+//losing message
+function lose(player, computer) {
+  playCount++;
+  lastWinner = "Computer";
+  return `You lost! You chose ${player}${cuteSelector(
+    player
+  )} and the computer chose ${computer}${cuteSelector(computer)}.
+  <br>${winRateMessage(winCount, playCount, userName)}`;
+}
+
+//draw message
+function draw(player, computer) {
+  playCount++;
+  lastWinner = "Draw";
+  return `Draw! You chose ${player}${cuteSelector(
+    player
+  )} and the computer chose ${computer}${cuteSelector(
+    computer
+  )}.<br>${winRateMessage(winCount, playCount, userName)} `;
+}
+
+//function to determine who wins the reversed game
+function playReversedGame(player, computer) {
+  if (player == computer) {
+    return draw(player, computer);
+  } else if (player == "scissors") {
+    if (computer == "stone") {
+      return win(player, computer);
+    } else if (computer == "paper") {
+      return lose(player, computer);
+    }
+  } else if (player == "paper") {
+    if (computer == "scissors") {
+      return win(player, computer);
+    } else if (computer == "stone") {
+      return lose(player, computer);
+    }
+  } else if (player == "stone") {
+    if (computer == "paper") {
+      return win(player, computer);
+    } else if (computer == "scissors") {
+      return lose(player, computer);
+    }
+  } else {
+    return "Please enter only scissors,paper or stone";
+  }
+}
+// function to vary the output message based on win rate
+function winRateMessage(wins, plays, name) {
+  let winrate = wins / plays;
+  let outputMessage = "";
```

Comment:
good attempt at using let keyword here. However, generally we would like to have a standardization of the syntax that we use through your code. Hence if you are declaring most of your variables with var, it would be good to keep it constant in your code. If you would like a challenge, you could try to use const and let in your project 2 and I can leave reviews for those

Reply:
Thanks for pointing it out. will keep it in mind for project 2 and keep things constant